src/pages/Exercise3.tsx

```diff
@@ -0,0 +1,98 @@
+import { FC, useState } from 'react';
+import '../styles/Exercise3.scss';
+import Pipi from '../../public/Pipi.svg';
+import AppWrapper from '../components/AppWrapper';
+import Box from '../components/Box';
+import Grid from '../components/Grid';
+import HintBox from '../components/HintBox';
+import NavButtons from '../components/NavButtons';
+import { HeaderSections } from '../types/globalTypes';
+
+const Exercise3: FC = () => {
+  const [confetti, setConfetti] = useState(false);
+  const nums = Array.from({ length: 24 }, (_, index) => index + 1);
+  const itemSpace = [1, 3, 1, 3, 2, 4, 2, 1, 1, 2, 1, 2, 1];
+  const selectedValue = '';
+  return (
+    <>
+      <AppWrapper section={HeaderSections.EXERCISE_3}>
+        <div className="page-wrapper">
+          <h1>Exercise 3</h1>
+          <p>
+            For this exercise, you will be helping Pipi find the basketball that
+            was stored in the last lesson and replacing it with a soccer ball.
+          </p>
+          <h2>Instructions</h2>
+          <p>
+            Take PiPi to <span className="highlight">Box I</span> by clicking on
```

Comment:
Let's change this to box J because the uppercase i and lowercase l are confusing....